api/queries_issue.go

```diff
@@ -106,10 +106,14 @@ type ProjectV2Item struct {
 		ID    string `json:"id"`
 		Title string `json:"title"`
 	}
-	Status struct {
-		OptionID string `json:"optionId" graphql:"... on ProjectV2ItemFieldSingleSelectValue{optionId}"`
-		Name     string `json:"name" graphql:"... on ProjectV2ItemFieldSingleSelectValue{name}"`
-	} `graphql:"status:fieldValueByName(name: \"Status\")"`
+	Status Status `graphql:"status:fieldValueByName(name: \"Status\")"`
+}
+
+type Status struct {
+	StatusFragment struct {
+		OptionID string `json:"optionId"`
+		Name     string `json:"name"`
+	} `graphql:"... on ProjectV2ItemFieldSingleSelectValue"`
```

Comment:
Nothing blocking this review, but I can't help but feel like this use case should be documented as part of the shurcool repository.  Just making a note for myself as I'm motivated to do it.

Reply:
I _think_ it's reflected in https://github.com/shurcooL/graphql#inline-fragments:

```
type (
	DroidFragment struct {
		PrimaryFunction graphql.String
	}
	HumanFragment struct {
		Height graphql.Float
	}
)

var q struct {
	Hero struct {
		Name          graphql.String
		DroidFragment `graphql:"... on Droid"`
		HumanFragment `graphql:"... on Human"`
	} `graphql:"hero(episode: \"JEDI\")"`
}
```